src/main/java/com/sulphate/chatcolor2/listeners/ColorGUIListener.java

```diff
@@ -78,7 +74,6 @@ else if (!colChar.equals(colour)) {
                     for (String modifier : modifiers) {
                         if (modifier.equals("")) continue;
 
-                        Bukkit.broadcastMessage("Appending Modifier: " + modifier);
```

Comment:
About the broadcastMessage call I've removed, this was it.

Reply:
Alright. Thanks for this. Final question, have you compiled this and tested it on a server?

Reply:
Yes, it compiles fine. It works on a 1.14.1 PaperMC server.